README.md

```diff
@@ -469,35 +465,43 @@ Fully customisable.  |
 :star2: | Name | Description | 🌍
 --- | --- | --- | ---
 2 | [@mkatircioglu](https://github.com/mkatircioglu)/[**dotfiles**](https://github.com/mkatircioglu/dotfiles) |  |
-### Crystal #
```

Comment:
I am sorry, I checked package.json file and I see that there is a duplicate of the list. Why do you have to put the complete list on the package.json again?